src/pageManager.js

```diff
@@ -706,14 +703,30 @@ export async function initialize() {
         }
     }
 
-    // Register the pageManager content script for all URLs permitted by the extension manifest
-    browser.contentScripts.register({
-        matches: permissions.getManifestOriginMatchPatterns(),
-        js: [{
-            file: pageManagerContentScript
-        }],
-        runAt: "document_start"
-    });
+    // Register the pageManager content script for all URLs permitted by the extension manifest.
+    // Only Firefox supports dynamic content script loading at this time.
+    try {
+        const browserInfo = await browser.runtime.getBrowserInfo();
+        if (browserInfo && browserInfo.name === "Firefox") {
+            browser.contentScripts.register({
+                matches: permissions.getManifestOriginMatchPatterns(),
+                js: [{
+                    file: pageManagerContentScript
+                }],
+                runAt: "document_start"
+            });
+        } else {
+            console.debug("WebScience pageManager loaded, requires content script:", pageManagerContentScript);
+        }
+    } catch (ex) {
+        if (ex.message === "browser.runtime.getBrowserInfo is not a function") {
+        console.debug("WebScience pageManager loaded, requires content script:", pageManagerContentScript);
```

Comment:
Nit: indentation.

Reply:
Thanks, I want to run `eslint --fix` on this repo, but I didn't want to introduce noise in the review... I'll do that in a separate commit.